apps/web/.eslintrc.js

```diff
@@ -22,8 +22,16 @@ module.exports = {
     'no-restricted-imports': [
       'error',
       {
-        paths: ['src'],
-        patterns: ['@novu/dal'],
+        paths: [
+          {
+            name: '@novu/dal',
+          },
+          {
+            name: '@mantine/core',
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This may be a pain, so we may have to revisit, but I think this will be the best way to prevent people from creating more debt of direct imports from Mantine‏

Reply:
Nice!

Reply:
Nice!